include/systems/inter_mesh_projection.h

```diff
@@ -42,7 +42,7 @@ namespace libMesh
     /**
      * This class implements inter mesh projection, i.e. projection of
      * vectors defined on a given mesh (from_mesh associated with from_system)
-     * to another mesh (to_mesh), stored in the map projected_vectors_map.
+     * to another mesh (to_mesh of to_system).
```

Comment:
This is another "probably ought to just be a fixup" commit.  We like having lots of intermediate working commits in the git history, because that helps pinpoint where breakage occurred when a new test fails, but broken commits don't help bisect anything.

I also wouldn't say "further testing completed" at this point; manually run test results aren't the same as unit tests.  A completed test is one that gets rerun with every "make check"; it's too easy to regress otherwise.